packages/kbn-test/src/auth/saml_auth.ts

```diff
@@ -51,6 +51,7 @@ const cleanException = (url: string, ex: any) => {
     ex.request = { REDACTED: 'REDACTED' };
     ex.response = { REDACTED: 'REDACTED' };
   }
+  return ex;
```

Comment:
__sugestion__: you don't use return value in https://github.com/elastic/kibana/pull/187988/files#diff-06f0d5181f1d9e35454c03de277b4d674e54bc83a603215eaf3066374dc12c17R174
so I believe we don't need it?
We thoughtfully modify the original error object, so there is no benefit of having it. Thoughts?

Reply:
No, that is me using it.  It mutates, but also returns.  
The order of operations will be that fn will evalute to it's value, first.

But I can change this to be less ambiguous, to follow up on your earlier comment.

Reply:
I think since we mutate there is no need to return it as you won't assign it to variable but rather just use the original object, right?

Reply:
yeah but mutation makes it harder to reason about.
the next push won't use mutation.